.gitignore

```diff
@@ -160,3 +160,6 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+
+*.pdf
+*.db
```

Comment:
This is pretty broad for the root .gitignore. I think you should add a .gitignore in recipe folders when necessary to ignore files created during notebook execution.